airflow/operators/sqlite_operator.py

```diff
@@ -31,6 +31,8 @@ class SqliteOperator(BaseOperator):
     :param sql: the sql code to be executed. (templated)
     :type sql: str or string pointing to a template file. File must have
         a '.sql' extensions.
+    :param parameters: (optional) the parameters to render the SQL query with.
+    :type parameters: mapping or iterable
```

Comment:
Could you fix the order? :) The `sql` one comes before `mssql_conn_id`, but in the docstring it is the other way around.